### ISSUE TITLE
On November 15th, skynetfree.net will be shutting down

### ISSUE THREAD

**Issue by RichardFevrier** [redsolver/vup#9]
Hello and thank you for all the work accomplished 🎉
I was wondering if [this](https://skynetfree.net/) gonna impact the project ?

**Comment by redsolver** [comment]
yes, I don't recommend using Vup right now. There's only one other Skynet portal remaining. I'm currently working on a new storage network that tries to fix most of the issues that caused Skynet to fail, see https://github.com/redsolver/S5. A new beta version of Vup that uses this new network should be ready in 1-2 months.
I recommend joining the Vup Discord Server for updates, it's also bridged to Matrix if you prefer that.

**Comment by trymeouteh** [comment]
> yes, I don't recommend using Vup right now. There's only one other Skynet portal remaining. I'm currently working on a new storage network that tries to fix most of the issues that caused Skynet to fail, see https://github.com/redsolver/S5. A new beta version of Vup that uses this new network should be ready in 1-2 months. I recommend joining the Vup Discord Server for updates, it's also bridged to Matrix if you prefer that.

Will Vup new storage network be sia network itself?! That would me amazing if it used sia and your able to also use vup as a sia wallet to manage storage contracts.

**Comment by redsolver** [comment]
v0.15.0 migrated to using the new S5 protocol and removed Skynet. native sia support and direct downloads/uploads from hosts are coming to S5 soon.
